src/cadenza.js

```diff
@@ -1104,7 +1104,7 @@ function array(/** @type unknown */ value) {
 /** @typedef {CadenzaEvent<'editGeometry:ok', {geometry: Geometry}>} CadenzaEditGeometryOkEvent - When the user submitted the geometry. */
 /** @typedef {CadenzaEvent<'editGeometry:cancel'>} CadenzaEditGeometryCancelEvent - When the user cancelled the geometry editing. */
 /** @typedef {CadenzaEvent<'error', {type: string, message?: string}>} CadenzaErrorEvent - An error event that is mapped to a {@link CadenzaError} */
-/** @typedef {CadenzaEvent<'objectInfo', {layer: WorkbookLayerPath, objectInfos: {selectionIndex: number, formattedValues: Record<string, string>}[]}>} CadenzaObjectInfoEvent - When the user opened the object info flyout. */
+/** @typedef {CadenzaEvent<'objectInfo', {layer: WorkbookLayerPath, objectInfos: {selectionIndex: number, elements: {attributePrintName: Record<string, string>, formattedValues: Record<string, string>}[]}}>} CadenzaObjectInfoEvent - When the user opened the object info flyout. */
```

Comment:
how is only a type improvement the fix for this bug? Shouldn't there be some change in code as well?

Reply:
The event is sent from Cadenza, i.e. the related code change is there.

Reply:
Like jkissel said.

The type was indeed wrong..

Reply:
It's still wrong, because it's only _one_ `formattedValue`.